.github/workflows/reusable_deploy_docs.yml

```diff
@@ -230,20 +238,3 @@ jobs:
         run: |
           git fetch
           python3 -m ghp_import -n -p -x docs/cpp/${{ inputs.CPP_DOCS_VERSION_NAME }} rerun_cpp/docs/html/ -m "Update the C++ docs"
-
-  update-rerun-io-env:
```

Comment:
This job is no longer needed, because we now fetch the latest release version directly in `on_push_docs`.